src/vs/workbench/workbench.desktop.main.ts

```diff
@@ -38,8 +38,8 @@ import 'vs/workbench/services/textfile/electron-sandbox/nativeTextFileService';
 import 'vs/workbench/services/dialogs/electron-sandbox/fileDialogService';
 import 'vs/workbench/services/workspaces/electron-sandbox/workspacesService';
 import 'vs/workbench/services/menubar/electron-sandbox/menubarService';
-import 'vs/workbench/services/issue/electron-sandbox/issueMainService';
-import 'vs/workbench/services/issue/electron-sandbox/issueService';
+import 'vs/workbench/contrib/issue/electron-sandbox/issueMainService';
+import 'vs/workbench/contrib/issue/electron-sandbox/issueService';
```

Comment:
There already is `vs/workbench/contrib/issue/electron-sandbox/issue.contribution` below in this file, I would suggest to remove these 2 lines and move them into the `issue.contribution` file.